global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "6.0.100-rc.1.21415.3",
+    "dotnet": "6.0.100-rc.2.21420.30",
```

Comment:
should we put dependency on this version? 
(like for runtime below) 

This way we always ensure that we run on n-1 SDK version. I think it would be beneficial for tests.

Reply:
Is there a dependency version for SDK? Please let me know if that's possible, and I will change this.

Reply:
I added a new dependency to installer in the last commit.